AdminPortalTest1stPage.java

```diff
@@ -0,0 +1,292 @@
+package Admin_Portal;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import webdriverbase.AppPage;
+
+public class AdminPortalTest1stPage extends AppPage {
+
+	public AdminPortalTest1stPage(WebDriver driver) {
+		super(driver);
+		
+	}
+	
+	
+	@FindBy(xpath = "//span[@class='hf-top-bar_title_text hf-font-light']")
+	private WebElement Title;
+	
+	@FindBy(linkText = "Statuses")
+	private WebElement Statuses;
+	
+	public void clickStatus()
+ 	{
+		hoverOverElementUsingJS(Title);
+		Title.click();
+		Statuses.click();
+		
+ 	}
+	
+	@FindBy(linkText = "Priorities")
+	private WebElement Priorities;
+	
+	public void clickPriorities() throws InterruptedException
+ 	{
+		sleep();
+		hoverOverElementUsingJS(Title);
+		Title.click();
+		hoverOverElementUsingJS(Priorities);
+		Priorities.click();

```

Comment:
No error handling in place for most of the methods implemented here
we can avoid throws exception